tests/__fixtures__/php-tags/php-on-same-split-line.php.php

```diff
@@ -0,0 +1,5 @@
+<?php echo "Some to be happy about"; ?>
+----
+@php
+ echo "Some to be happy about";
+@endphp
```

Comment:
Please add a newline at the end of the file
```suggestion
@endphp

```

Reply:
This adds extra breakage to the tests